### DISCUSSION TITLE
is there a way to parse a compact number into a real number?

### DISCUSSION THREAD

**Discussion by MatthewRack** [python-babel/babel#1073] [Q&A]
I know how to go from 54,123 to 54K using format_compact_decimal. Is there a way to go from 54K to 54,000 for any locale the compact number might be in?

**Comment by akx** [comment]
There is no such function in Babel at present.

If you can find an Unicode specification for parsing them ([as there is for formatting](https://cldr.unicode.org/translation/number-currency-formats/number-and-currency-patterns#h.eradhhuxzqqz)) we can look into implementing that.
